### PR TITLE
Add RoughCanvas

### LINKED ISSUE #123: Ellipse method is missing
The ellipse method is missing from ipycanvas.

https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/ellipse

### PR DESCRIPTION
**Pull request by martinRenou**
# TODO

- [ ] Add more options
- [ ] Support Ellipse?
- [x] Docs
- [x] Add support for Path?

# Fixes

This PR happens to fix https://github.com/martinRenou/ipycanvas/issues/91 and fixes #123,  it's a partial solution for https://github.com/martinRenou/ipycanvas/issues/27

sketchy

### CODE REVIEW

**Comment by matthewturk** [comment]
This is awesome!

**Comment by martinRenou** [comment]
Wow thanks for your message!

I am not sure about the name for the canvas class though, would you have any comment on it? Any idea for a better name if you think it's not quite good?

File: ipycanvas/canvas.py [hunk 1]
```diff
@@ -612,6 +625,37 @@ def _handle_frontend_event(self, _, content, buffers):
...
+    """
+
+    _model_name = Unicode('SketchyCanvasModel').tag(sync=True)
+    _view_name = Unicode('CanvasView').tag(sync=True)
+
+    #: (str) Sets the appearance of the filling, possible values are ``'hachure'``, ``'solid'``, ``'zigzag'``,
+    #: ``'cross-hatch'``, ``'dots'``, ``'sunburst'``, ``'dashed'``, ``'zigzag-line'``.
+    #: Default to ``'hachure'``.
+    sketchy_fill_style = Enum(['hachure', 'solid', 'zigzag', 'cross-hatch', 'dots', 'sunburst', 'dashed', 'zigzag-line'], default_value='hachure')
+
```
Comment (matthewturk) [inline ipycanvas/canvas.py:644]: Would you be willing to add an option for the `roughness` parameter for roughjs?

Comment (martinRenou) [reply ipycanvas/canvas.py:644]: Definitely! Thanks

**Comment by matthewturk** [comment]
@martinRenou I'm cool with `SketchyCanvas` but I have a history of terrible naming decisions, so ... :man_shrugging:

File: ipycanvas/canvas.py (hunk 1 above)
Comment (martinRenou) [reply ipycanvas/canvas.py:644]: Done!

**Comment by deeplook** [comment]
RoughCanvas?

**Comment by martinRenou** [comment]
@deeplook That was the first option, I thought `SketchyCanvas` sounded better maybe?

**Comment by stonebig** [comment]
if ever there is a sudden synergy between ipycanvas and Wacom users, maybe shall you keep the sketchy word for truly sketchy practice ?

**Comment by martinRenou** [comment]
Ok, let's go for `RoughCanvas` then!

Thanks everyone :)

**Comment by martinRenou** [comment]
Let's call it done :)

**Pull request merged**